app/layouts/default-layout.tsx

```diff
@@ -33,6 +34,10 @@ const DefaultLayout: React.FunctionComponent<Props> = ({
   width = 'narrow',
   help
 }) => {
+  const isInternalUser =
+    session?.userGroups.includes('Incentive Administrator') ||
+    session?.userGroups.includes('Incentive Analyst');
```

Comment:
```ts
import {INCENTIVE_ANALYST, ADMIN_GROUP} from 'data/group-constants';

const internalUserGroups = [INCENTIVE_ANALYST, ...ADMIN_GROUP];
```

You can retrieve the internal user groups like this (you're missing the "Realm Administrator" group), and loop through one array to see if there is an intersection between both groups.